Smartling.API/Smartling.Api.csproj

```diff
@@ -7,6 +7,8 @@
     <Version>2.4.34.0</Version>
     <Authors>Smartling</Authors>
     <Copyright>Copyright © Smartling 2019</Copyright>
+    <SignAssembly>false</SignAssembly>
+    <AssemblyOriginatorKeyFile>Key1.snk</AssemblyOriginatorKeyFile>
```

Comment:
Unrelated change. SDK must not know about some keys etc

Reply:
Remove